inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicWebController.java

```diff
@@ -61,47 +68,188 @@ public class TopicWebController {
+     */
+    private TubeMQResult handleModifyTopicRequest(String req) {
+        return masterService.baseRequestMaster(gson.fromJson(req, ModifyTopicReq.class));
+    }
+
+    /**
+     * Handles 'DELETE' and 'REMOVE' method requests.
+     *
+     * @param req The JSON request
+     * @return TubeMQResult based on the request
+     */
+    private TubeMQResult handleDeleteTopicRequest(String req) {
+        return masterService.baseRequestMaster(gson.fromJson(req, DeleteTopicReq.class));
+    }
+
+    /**
+     * Handles 'QUERY_CAN_WRITE' method request.
+     *
+     * @param req The JSON request
+     * @return TubeMQResult based on the request
+     */
+    private TubeMQResult handleQueryCanWriteRequest(String req) {
+        QueryCanWriteReq queryCanWriteReq = gson.fromJson(req, QueryCanWriteReq.class);
+        if (!queryCanWriteReq.legal()) {
             return TubeMQResult.errorResult(TubeMQErrorConst.PARAM_ILLEGAL);
         }
-        return topicService.queryCanWrite(req.getTopicName(), req.getClusterId());
+        return topicService.queryCanWrite(queryCanWriteReq.getTopicName(), queryCanWriteReq.getClusterId());
+    }
+
+    /**
+     * Handles 'PUBLISH' method request.
+     *
+     * @param req The JSON request
+     * @return TubeMQResult based on the request
+     */
+    private TubeMQResult handlePublishRequest(String req) {
+        return masterService.baseRequestMaster(gson.fromJson(req, SetPublishReq.class));
+    }
+
+    /**
+     * Handles 'SUBSCRIBE' method request.
+     *
+     * @param req The JSON request
+     * @return TubeMQResult based on the request
+     */
+    private TubeMQResult handleSubscribeRequest(String req) {
+        return masterService.baseRequestMaster(gson.fromJson(req, SetSubscribeReq.class));
+    }
+
+    /**
+     * Checks if the given method is valid by comparing it against a list of allowed methods.
+     *
+     * @param method The method to validate.
+     * @return {@code true} if the method is valid, {@code false} otherwise.
+     */
+    private static boolean isValidMethod(String method) {
+        // Define a list of allowed methods
+        List<String> allowedMethods = Arrays.asList(
+                TubeConst.ADD, TubeConst.CLONE, TubeConst.AUTH_CONTROL, TubeConst.MODIFY,
+                TubeConst.DELETE, TubeConst.REMOVE, TubeConst.QUERY_CAN_WRITE, TubeConst.PUBLISH, TubeConst.SUBSCRIBE);
+        return allowedMethods.contains(method);
+    }
+
+    /**
+     * Validates whether the given JSON string has a valid format by attempting to parse it.
+     *
+     * @param json The JSON string to validate.
+     * @return {@code true} if the JSON format is valid, {@code false} otherwise.
+     */
+    private static boolean isValidJson(String json) {
+        // Use a JSON library or parser to validate the JSON format
+        try {
+            gson.fromJson(json, Object.class);
+            return true;
+        } catch (JsonSyntaxException e) {
+            return false;
```

Comment:
Better to log these exceptions in debug, warn, or error level, depending on their importance.

Reply:
Thanks for the correction, have added the appropriate logs to record exceptions